### PR TITLE
Customised passport

### PR DESCRIPTION
**Pull request by allanchau**
This PR adds the ability to customise the linz authenticate middleware.

**Testing**

- [x] Include this branch into your project.

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -19,23 +19,35 @@ const whoAmI = require('./routes/who-am-i');
...
 
         linz.init({
-            'express': app,
-            'options': {
+            express: app,
+            options: {
+                'login middleware': {
+                    get: [linz.api.middleware.login.get],
+                    post: [
+                        linz.api.middleware.authenticate('linz-local', {
```
Comment (smebberson) [inline app/app.js:40]: @allanchau, do you think this should be available at `linz.api.middleware.login.post`?

**Comment by smebberson** [comment]
@allanchau, could you please also update/add to the documentation for this change?

File: app/app.js (hunk 1 above)
Comment (allanchau) [reply app/app.js:40]: @smebberson I've moved this bit of code to `linz.api.middleware.login.post` now.

**Comment by allanchau** [comment]
@smebberson I don't think so, it should work fine with all current projects. We just added the ability to customise the login.

**Pull request merged**